hub/models.py

```diff
@@ -29,9 +29,33 @@
 from strawberry.dataloader import DataLoader
 from pyairtable import Api as AirtableAPI, Base as AirtableBase, Table as AirtableTable
 
+
 User = get_user_model()
 
 
+class Organisation(models.Model):
+    slug = models.SlugField(max_length=100, unique=True)
+    name = models.CharField(max_length=100)
+    description = models.TextField(blank=True, null=True)
+    website = models.URLField(blank=True, null=True)
```

Comment:
I think django recommend using blank=True and null=False for fields that return strings: https://docs.djangoproject.com/en/5.0/ref/models/fields/#null